ReflectionAnalyzers.Vsix/source.extension.vsixmanifest

```diff
@@ -1,30 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Id="ReflectionAnalyzers.Vsix.891942ac-38e5-4220-b8d7-a96617a1bd07" Version="1.0" Language="en-US" Publisher="Johan Larsson" />
-    <DisplayName>ReflectionAnalyzers.Vsix</DisplayName>
+    <Identity Id="ReflectionAnalyzers.891942ac-38e5-4220-b8d7-a96617a1bd07" Version="1.0" Language="en-US" Publisher="Johan Larsson" />
+    <DisplayName>ReflectionAnalyzers</DisplayName>
     <Description>ReflectionAnalyzers</Description>
   </Metadata>
 
   <Installation>
-    <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.Pro" />
-    <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.VSWinDesktopExpress" />
-    <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.VWDExpress" />
-    <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.VSWinExpress" />
+    <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.Community" />
   </Installation>
 
-  <Dependencies>
-    <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="4.5" />
-  </Dependencies>
+  <Prerequisites>
+    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[14.0,)" DisplayName="Visual Studio core editor" />
+    <Prerequisite Id="Microsoft.VisualStudio.Component.Roslyn.LanguageServices" Version="[14.0,)" DisplayName="Roslyn Language Services" />
+  </Prerequisites>
 
   <Assets>
     <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="ReflectionAnalyzers" Path="|ReflectionAnalyzers|"/>
     <Asset Type="Microsoft.VisualStudio.Analyzer" d:Source="Project" d:ProjectName="ReflectionAnalyzers" Path="|ReflectionAnalyzers|"/>
     <Asset Type="Microsoft.VisualStudio.Analyzer" d:Source="File" Path="Gu.Roslyn.Extensions.dll" />
     <Asset Type="Microsoft.VisualStudio.Assembly" d:Source="File" Path="Gu.Roslyn.Extensions.dll" />
   </Assets>
-
-  <Prerequisites>
-    <Prerequisite Id="Microsoft.VisualStudio.Component.Roslyn.Compiler" Version="[15.0.26208.0,16.0)" DisplayName="C# and Visual Basic Roslyn compilers" />
```

Comment:
@JohanLarsson I'm hesitant to delete this. Do you know why this was set (and disagreed with the other 14.0 constraints)?

Reply:
/cc @sharwell if you have time

Reply:
Try removing it and see if it works. Most likely that line is me cargo culting.

Reply:
The only way to see the difference is to try various versions of VS2015. Should we bump up the minimum version to 2017 or 2019, or just remember to revisit if you ever take things further and publish to the VS gallery?